packages/commonwealth/client/scripts/views/components/component_kit/cw_avatar_group.tsx

```diff
@@ -48,11 +51,15 @@ export const CWAvatarGroup = (props: AvatarGroupProps) => {
               return addr.chain == chainId;
             });
 
-            return (
-              <div className="avatar-group-icon" key={i}>
-                <CWJdenticon address={address.address} size={16} />
-              </div>
-            );
+            // some old posts are broken = have no address in the specified community.
+            // if so, we omit their icon as a quick fix.
+            if (address) {
+              return (
+                <div className="avatar-group-icon" key={i}>
+                  <CWJdenticon address={address.address} size={16} />
+                </div>
+              );
+            }
```

Comment:
I think it would be good to add a fallback if there is no address. Is there a possibility for all addresses to be broken in `truncatedProfiles`?

Reply:
@mzparacha the idea was it would simply render nothing -- but I can make that explicit.

Reply:
Should be fixed. Take another look @mzparacha